src/Ajax_Inspector/Plugin.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Plugin class file
+ */
+
+namespace SDOKUS\Ajax_Inspector;
+
+// Make sure that the file is being accessed in the WP environment.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Make sure that the class is not being duplicated.
+if ( ! class_exists( 'Dokus__Ajax__Inspector' ) ) {
+	/**
+	 * The Ajax Inspector Class
+	 *
+	 * This is the main class of the plugin
+	 */
+	class Dokus__Ajax__Inspector {
+
+		/**
+		 * Static Singleton Holder
+		 *
+		 * @var self
+		 */
+		private static $instance = null;
+
+		/**
+		 * Get (and instantiate, if necessary) the instance of the class
+		 *
+		 * @return self
+		 */
+		public static function get_instance() {
+			if ( ! self::$instance ) {
+				self::$instance = new Dokus__Ajax__Inspector();
+			}
+
+			return self::$instance;
+		}
+
+		/**
+		 * Initializes plugin variables and sets up WordPress hooks/actions.
+		 */
+		protected function __construct() {
+			$this->enable_hooks();
+
+            do_action( 'ajax_plugin_loaded');
+		}
+
+		/**
+		 * Enable hooks for the plugin
+		 */
+		public function enable_hooks() {
+			add_action( 'ajax_plugin_loaded', [$this, 'load_assets'] );
+ //        add_action( 'wp_enqueue_scripts', [ $this, 'add_jquery' ] );
+//			add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_ajax_button_script' ] );
+			add_shortcode( 'ajax_button', [ $this, 'ajax_button_shortcode' ] );
+		}
+
+		/**
+		 * Disable hooks for the plugin
+		 */
+		public function disable_hooks() {
+//			remove_action( 'wp_enqueue_scripts', [ $this, 'add_jquery' ] );
+//			remove_action( 'wp_enqueue_scripts', [ $this, 'enqueue_ajax_button_script' ] );
+			remove_shortcode( 'ajax_button' );
+		}
+
+
+//		/**
+//		 * Enqueues jquery
+//		 *
+//		 * @return void
+//		 */
+//		public function add_jquery(): void {
+//			wp_enqueue_script( 'jquery' );
+//		}
+//
+//		/**
+//		 * Enqueues the custom JS script for the button
+//		 *
+//		 * @return void
+//		 */
+//		public function enqueue_ajax_button_script(): void {
+//			// Enqueue the CSS file.
+//			wp_enqueue_style(
+//				'style',
+//				plugin_dir_url( __FILE__ ) . '../resources/css/style.css',
+//				[],
+//				'1.0'
+//			);
+//
+//            // Enqueue the JS script.
+//            wp_enqueue_script(
+//				'ajax-button-script',
+//				plugin_dir_url( __FILE__ ) . '../resources/js/ajax-button-script.js',
+//				[ 'jquery' ],
+//				'1.0',
+//				true
+//			);
+//			// Localize script with nonce to MyAjax object
+//			wp_localize_script('ajax-button-script', 'ajax_button_script_vars', array(
+//				'ajaxurl' => admin_url('admin-ajax.php'), // This line localizes the 'ajaxurl' variable
+//			));
+//		}
```

Comment:
When using a version control you dont need to comment out code. As long as you committed the older version you will always have access to the history to go back to.